documentation/2.0/education.md

```diff
@@ -40,6 +44,7 @@ Here is the list of [default content fields](../../schemas/2.0/text-property.xsd
 |`degree`|The certification that the participants may receive|
 |`continuing`|How to continue the studies after this course (i.e. advanced level etc.)|
 |`detailedCost`|Detailed information about pricing and what’s included in the price|
+|`*Training Course Content*`|Detailed information about course content (i.e. topics, modules, syllabus)|
```

Comment:
Default content fields cannot be modified. I understand your need but I think we need to use a different strategy for this.